examples/linked.rs

```diff
@@ -0,0 +1,37 @@
+use web3::api::Web3;
+use web3::transports::Http;
+
+ethcontract::contract!("examples/truffle/build/contracts/SimpleLibrary.json");
+ethcontract::contract!("examples/truffle/build/contracts/LinkedContract.json");
+
+fn main() {
+    futures::executor::block_on(run());
+}
+
+async fn run() {
+    let (eloop, http) = Http::new("http://localhost:9545").expect("transport failure");
+    eloop.into_remote();
+    let web3 = Web3::new(http);
+
+    let library = SimpleLibrary::deploy(&web3)
+        .gas(4_712_388.into())
+        .confirmations(0)
+        .deploy()
+        .await
+        .expect("library deployment failure");
+    let instance = LinkedContract::deploy(&web3, library.address())
```

Comment:
How do argument names for the constructor work. Do they come after or before the linked libraries?

Reply:
Good point! Maybe I should add this to the example.